test/correctness/CMakeLists.txt

```diff
@@ -169,6 +169,7 @@ tests(GROUPS correctness travis
         inverse.cpp
         isnan.cpp
         issue_3926.cpp
+        issue_4297.cpp
```

Comment:
see several previous comments about this filename

Reply:
Fixed!